modules/calendar/add_edit_event.php

```diff
@@ -230,6 +230,15 @@ function DOBandEncounter()
         if ($_POST['form_ampm'] == '2' && $tmph < 12) $tmph += 12;
         $duration = abs($_POST['form_duration']); // fixes #395
     }
+       
+     if($tmph*60+$tmpm > $GLOBALS['schedule_end']*60 or $tmph < $GLOBALS['schedule_start']){
+        echo "<html>\n<body>\n<script language='JavaScript'>\n";
+        echo 'alert("Please select time between clinic hours.");';
```

Comment:
xlt()

Reply:
I will put a global to turn this feature on and off as suggested by Teryhill.